modules/microshift-low-latency-tuned-auto.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-tuned-auto-activate_{context}"]
+= Automatically enable the {microshift-short} TuneD profile
+
+Included in the `microshift-low-latency` RPM package is a systemd service that you can configure to automatically enable a TuneD profile when the system starts. This ability is particularly useful if you are installing {microshift-short} in a large fleet of devices.
+
+.Prerequisites
+
+. You installed the microshift-low-latency RPM package on the host.
+. You enabled low latency in the {microshift-short} `config.yaml`.
+. You created a TuneD profile.
+
+.Procedure
+
+. Configure the `tuned.yaml` in the `/etc/microshift/` directory, for example:
++
+.Example tuned.yaml
+[source,yaml]
+----
+# ...
+profile: microshift-baseline <1>
+reboot_after_apply: True <2>
+# ...
+----
+<1> The `profile` value controls which TuneD profile is activated. In this example, the name of the profile is `microshift-baseline`.
+<2> The `reboot_after_apply` value controls whether the host must be rebooted after applying the profile. Valid values are `True` and `False`. For example, use the `True` setting to automatically restart the host after a new `ostree` commit is deployed. Changes to the contents of the TuneD profile or variables are detected by the `microshift-tuned.service` and trigger reactivation of the profile, then reboots the host.
```

Comment:
More precisely, it restarts the host when `microshift-tuned.service` runs (and usually that's on boot).
When new commit is deployed, the system needs to rebooted somehow, but `microshift-tuned` is not involved here.
But when that happens there will be two reboots in a row:
- First to enable new commit
- Second when `microshift-tuned` starts on boot and sees that profile/variables changed.

I think we should mention that this will affect greenboot - specifically there's configurable amount of reboots it can make before rolling back, and when using this automatic profile activation, one will be used so it needs to be accounted for (like increase the num of boots from 3 to 4)